### PR TITLE
style: let user know how to input GitHub repository

### PR DESCRIPTION
**Pull request by jaumefapa**
### Description

No relevant changes. Just adding extra information in a string so the user knows how to  input content in the CLI.
My first time using Firebase and I had to google how to input the GitHub repo. I thought might be the same for others :)

### Scenarios Tested

None. I just changed a sentence.

### CODE REVIEW

File: src/init/features/hosting/github.ts [hunk 1]
```diff
@@ -392,7 +392,7 @@ async function promptForRepo(
       type: "input",
       name: "repo",
       default: defaultGithubRepo(), // TODO look at github origin
-      message: "For which GitHub repository would you like to set up a GitHub workflow?",
+      message: "For which GitHub repository would you like to set up a GitHub workflow? (username/repositoryName)",
```
Comment (samtstern) [inline src/init/features/hosting/github.ts:395]:
```suggestion
      message: "For which GitHub repository would you like to set up a GitHub workflow? (format: user/repository)",
```

**Comment by samtstern** [comment]
@jaumefapa thanks for the PR!  I made one small suggestion, and I think running `npm run format` should clear up the CI issues.

**Review by samtstern — APPROVED** [review]

File: src/init/features/hosting/github.ts [hunk 2]
```diff
@@ -392,7 +392,7 @@ async function promptForRepo(
       type: "input",
       name: "repo",
       default: defaultGithubRepo(), // TODO look at github origin
-      message: "For which GitHub repository would you like to set up a GitHub workflow?",
+      message: "For which GitHub repository would you like to set up a GitHub workflow? (format: user/repository)",
```
Comment (samtstern) [inline src/init/features/hosting/github.ts:395]: There's some lint failure on this line.

Comment (jaumefapa) [reply src/init/features/hosting/github.ts:395]: I'm staring at the line and I've no clue which linting problem could it be... the only idea I can come with is that CLI message have length constraints but, that doesn't make much sense...
Anyway, not a big deal the contribution. We can drop it :(

**Comment by samtstern** [comment]
@jaumefapa I checked out the branch locally, here's the change:

```diff
diff --git a/src/init/features/hosting/github.ts b/src/init/features/hosting/github.ts
index dc88ae9d..2551596d 100644
--- a/src/init/features/hosting/github.ts
+++ b/src/init/features/hosting/github.ts
@@ -392,7 +392,8 @@ async function promptForRepo(
       type: "input",
       name: "repo",
       default: defaultGithubRepo(), // TODO look at github origin
-      message: "For which GitHub repository would you like to set up a GitHub workflow? (format: user/repository)",
+      message:
+        "For which GitHub repository would you like to set up a GitHub workflow? (format: user/repository)",
       validate: async (repo: string) => {
         const { body } = await api.request("GET", `/repos/${repo}/actions/secrets/public-key`, {
           origin: api.githubApiOrigin,
```

**Comment by jaumefapa** [comment]
Hi @samtstern, locally I have:
1. Fixed the line break
2. Ran 'npm run format'
3. Committed and push to my repo
But still not passing CI tests. 
The truth is I still struggle with Git conflicts... can you fix it somehow? This linting is killing me. 
Cheers!

**Comment by samtstern** [comment]
@jaumefapa yep no problem

**Comment by samtstern** [comment]
@jaumefapa merged, thanks for the PR!

**Comment by jaumefapa** [comment]
Thanks for the help @samtstern! You avoid me a headache! 💊

**Pull request merged**